src/coreclr/vm/readytoruninfo.cpp

```diff
@@ -706,6 +725,17 @@ ReadyToRunInfo::ReadyToRunInfo(Module * pModule, LoaderAllocator* pLoaderAllocat
         m_entryPointToMethodDescMap.Init(TRUE, &lock);
     }
 
+    if (IsImageVersionAtLeast(6, 2))
```

Comment:
Why do we need to check the R2R format version? I was under the impression that we reject any older R2R file upfront so we don't even get here.